src/riak_cs_block_server.erl

```diff
@@ -270,6 +271,9 @@ do_get_block(ReplyPid, Bucket, Key, ClusterID, UseProxyGet, ProxyActive,
                   Timeout, ProceedFun, RetryFun, NumRetries, UseProxyGet,
                   ProxyActive, ClusterID).
 
+-spec try_local_get(riak_client(), binary(), binary(), list(), list(),
+                    timeout(), fun(), fun(), non_neg_integer(), boolean(),
+                    boolean(), binary()) -> no_return().
```

Comment:
The doc[1] says `no_return()` = `none()` is bottom type. But this function can return `ok` or any term that `ProceedFun` returns. It should be `any()` or `term()`. 


[1] http://www.erlang.org/doc/reference_manual/typespec.html